public/main.js

```diff
@@ -292,6 +343,8 @@ let backToLogin = (event) => {
     let loginContainer = document.querySelector('.login-input-container');
     signupContainer.classList.add('hidden');
     loginContainer.classList.remove('hidden');
+    document.querySelector('.view-saved')
+            .classList.add('hidden');
```

Comment:
It seems like these two lines show up a few times. I guess its hard to visualize just by looking at the code, but I just wasn't sure if there was an opportunity to refactor somehow. This comment is probably more like a bookmark for us to revisit.